reference/constraints/NotPwned.rst

```diff
@@ -0,0 +1,133 @@
+NotCompromisedPassword
+======================
+
+.. versionadded:: 4.3
+
+    The ``NotCompromisedPassword`` constraint was introduced in Symfony 4.3.
+
+Validates that the given password has not been compromised by checking that is
+not included in any of the public data breaches tracked by `haveibeenpwned.com`_.
+
+==========  ===================================================================
+Applies to  :ref:`property or method <validation-property-target>`
+Options     - `groups`_
+            - `message`_
+            - `payload`_
+            - `skipOnError`_
+            - `threshold`_
+Class       :class:`Symfony\\Component\\Validator\\Constraints\\NotCompromisedPassword`
+Validator   :class:`Symfony\\Component\\Validator\\Constraints\\NotCompromisedPasswordValidator`
+==========  ===================================================================
```

Comment:
I'm not sure this syntax works as expected. Won't it render the list of options all in a new row, instead of one row with a list of options?

Reply:
We changed this a few weeks ago. It looks like the old verbose table format. See https://symfony.com/doc/current/reference/constraints/NotBlank.html